CDP4ShellDialogs/ViewModels/ModelSelectionIterationSetupRowViewModel.cs

```diff
@@ -47,10 +47,12 @@ public ModelSelectionIterationSetupRowViewModel(IterationSetup iterationSetup, P
 
             if (participant.Domain.Count != 0)
             {
-                this.DomainOfExpertises.AddRange(participant.Domain.OrderBy(x => x.ShortName)); 
+                this.DomainOfExpertises.AddRange(participant.Domain.OrderBy(x => x.ShortName));
 
-                this.SelectedDomain = this.DomainOfExpertises.Contains(this.ActiveParticipant.Person.DefaultDomain)
-                    ? this.ActiveParticipant.Person.DefaultDomain
+                var selectedDomain2 = this.ActiveParticipant.SelectedDomain.Name;
```

Comment:
I have the impression that `selectedDomain2` is set but never used, if that is indeed the case, please remove